react/components/molecules/keyboard/keyboard.js

```diff
@@ -10,6 +10,7 @@ export class Keyboard extends Component {
             supportedCharacters: PropTypes.string,
             symbolicKeyboard: PropTypes.bool,
             capsLock: PropTypes.bool,
+            animations: PropTypes.bool,

```

Comment:
perhaps something like `animate`? With this name it seems that it receives a set of animations